examples/Getting Started with Boutiques.ipynb

```diff
@@ -0,0 +1,351 @@
+    "    ],\n",
+    "    \"output-files\": [\n",
+    "        {TODO},\n",
+    "        {TODO},\n",
+    "        ...\n",
+    "    ],\n",
+    "    \"groups\": [\n",
+    "        {TODO},\n",
+    "        {TODO},\n",
+    "        ...\n",
+    "    ]\n",
+    "}\n",
+    "```\n",
+    "\n",
+    "As the file is beginning to grow considerably in number of lines, we will no longer show you the full JSON at each step but will simply show you the dictionaries responsible for output, input, and group entries. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Step 3.2.1: Specifying inputs\n",
+    "\n",
+    "The input schema contains several options, many of which can be ignored with the exception of `id`, `name`, and `type`. These keys can be seen enumerated in the [documentation](http://boutiques.github.io/docs). For BET, there are several input values we can choose to demonstrate this for you. We have chosen three with considerably different functionality and therefore schemas. In particular:\n",
+    "- `[INPUT_FILE]`\n",
+    "- `[FRACTIONAL_INTENSITY]`\n",
+    "- `[CENTER_OF_GRAVITY]`\n",
+    "\n",
+    "**`[INPUT_FILE]`**   The simplest of these in the `[INPUT_FILE]` which is a required parameter that simply expects a qualified path to a file. The dictionary entry is:\n",
+    "```\n",
+    "{\n",
+    "    \"id\" : \"infile\",\n",
+    "    \"name\" : \"Input file\",\n",
+    "    \"type\" : \"File\",\n",
+    "    \"description\" : \"Input image (e.g. img.nii.gz)\",\n",
+    "    \"optional\": false,\n",
+    "    \"command-line-key\" : \"[INPUT_FILE]\"\n",
+    "}\n",
+    "```\n",
+    "\n",
+    "**`[FRACTIONAL_INTENSITY]`**   This parameter documents an optional flag that can be passed to the executable. Along with the flag, when it is passed, is a floating point value that can range from 0 to 1. We are able to validate at the level of Boutiques whether or not a valid input is passed, so that jobs are not submitted to the scheduler which will error, but they get flagged upon validation of inputs. This dictionary is:\n",
+    "```\n",
+    "{\n",
+    "    \"id\" : \"fractional_intensity\",\n",
+    "    \"name\" : \"Fractional intensity threshold\",\n",
+    "    \"type\" : \"Number\",\n",
+    "    \"description\" : \"Fractional intensity threshold (0->1); default=0.5; smaller values give larger brain outline estimates\",\n",
+    "    \"command-line-flag\": \"-f\",\n",
+    "    \"optional\": true,\n",
+    "    \"command-line-key\" : \"[FRACTIONAL_INTENSITY]\",\n",
+    "    \"integer\" : false,\n",
+    "    \"minimum\" : 0,\n",
+    "    \"maximum\" : 1\n",
+    "}\n",
+    "```\n",
+    "\n",
+    "**`[CENTER_OF_GRAVITY]`**   The center of gravity value expects a triple (i.e. [X, Y, Z] position) if the flag is specified. Here we are able to set the condition that the length of the list received after this flag is 3, by specifying both the minimum and maximum length.\n",
```

Comment:
-> by specifying *that the input is a list that has both a minimum and a maximum length.*